how-to/install-full-node.md

```diff
@@ -13,22 +13,18 @@ The quickest way is to simply download the binary files from the link below and
 ```bash
 wget https://echo-bin.s3.eu-central-1.amazonaws.com/echo-linux.v0.8.2.tar
 tar -xvf echo-linux.v0.8.2.tar
-cd echo-linux.v0.8.2.tar
-./echo_node \
-    --seed-node=echo-testnet-us-1.echo-dev.io:6310  \
-    --seed-node=echo-testnet-eu-1.echo-dev.io:6310 \
-    --rpc-endpoint=127.0.0.1:6312 --testnet
+cd echo-linux.v0.8.2
```

Comment:
If we remove the launch information, maybe it will be correct to add a link to the launch instruction page?